jscomp/syntax/src/res_scanner.ml

```diff
@@ -245,25 +251,30 @@ let scanNumber scanner =
         8)
     | _ -> 10
   in
-  scanDigits scanner ~base;
+  ignore (scanDigits scanner ~base);
```

Comment:
not a big fan of ignore here, it would swallow errors if we add an argument to the function later on
```suggestion
  let _foundDigits: bool = scanDigits scanner ~base;
```
But it's likely something we should instead fix on master.

Reply:
@tsnobip Agreed - would you do a PR on master?

Reply:
@cknitt done here 
https://github.com/rescript-lang/rescript/pull/7280